src/shared/backend/replicon_channels.rs

```diff
@@ -39,7 +39,7 @@ impl Default for RepliconChannels {
 
 impl RepliconChannels {
     /// Creates a new server channel and returns its ID.
-    pub(crate) fn create_server_channel(&mut self, channel: Channel) -> usize {
+    pub fn create_server_channel(&mut self, channel: Channel) -> usize {
```

Comment:
Do you want to create custom channels? Why not use the backend for this? :thinking: You'd have more configuration options that way.
This struct is for channels utilized by Replicon.

Reply:
Ok